gammapy/background/off_data_background_maker.py

```diff
@@ -203,3 +211,59 @@ def save_models(self, modeltype):
         """
         for ngroup in range(self.ntot_group):
             self.save_model(modeltype, ngroup)
+
+    def background_links(self, data_store, modeltype, out_dir_background_model=None, outfile=None,
+                            filename_obs_group_table=None):
+        """Creates a link for each observation of the obs_table to the corresponding background model
```

Comment:
From this docstring it's not obvious to me what the function does.

I think it just writes a file and doesn't modify data store members like the HDU index table.
Is that right?
Can you state in the docstring what it does?